src/test/java/LoginTests.java

```diff
@@ -15,11 +17,25 @@ public void LoginEmptyEmailPasswordTest() {
         options.addArguments("--remote-allow-origins=*");
 
         WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
         String url = "https://qa.koel.app/";
         driver.get(url);
         Assert.assertEquals(driver.getCurrentUrl(), url);
+
+        WebElement registration = driver.findElement(By.xpath("//div/a"));
```

Comment:
Hi,
I have checked your homework 16, you're doing great! Keep up the good work!
Best wishes from the HW team (Khanh, khanh@testpro.io)

Test script PASSED
![image](https://github.com/olehtytarenko/A47-QA-Automation-Framework/assets/144403954/e80974d5-98cd-4ac6-a97e-41f18e3c22dd)

But I have a few suggestions for you as follows:
- WebElement registration you can define with specific attributes for easier identification. 
Example:  WebElement registration = driver.findElement(By.xpath("//a[@href='registration']"));
- Next time, please help to create script with name homework (example: Homework16). It will be easier for us to look which script I need to review. 
Best wishes from the HW team (Khanh, khanh@testpro.io)